app/components/dds/dds-file-picker.js

```diff
@@ -6,14 +6,26 @@ const DDSFilePicker = Ember.Component.extend({
   children: null, // Can be files or folders
   onPick: (/* file */) => {},
   selectedResources: null,
-  fileFilter: (/*item, index, enumerable*/) => {return true},
+  formatSettings: {},
```

Comment:
One suggestion about convention here, since I learned it the hard way. Lots of scare quotes ahead :)

In Ember, providing an Object (or Array) value for a property in this definition makes it shared among all "instances" of the "class". I'm assuming that's not what you intended, since you're overwriting it as an "instance variable" in the hbs files.

If there needs to be a default value per instance, that can be done in `init()`. But if not, setting it to `null` is a good hint.

Reply:
That is weird.